Gemfile

```diff
@@ -1,7 +1,7 @@
 source "https://rubygems.org"
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
-ruby File.read(".ruby-version").chomp
+ruby "2.7.1"
```

Comment:
Why was this hardcoded?

Reply:
My laziness. When build failed and told me it couldn't find the file, I hardcoded it (took that from the other repo). 

But adding a line to copy `.ruby-version` file fixed it :)